azure/template.json

```diff
@@ -27,16 +27,68 @@
         },
         "logAnalyticsWorkspaceName": {
             "type": "string"
+        },
+        "containerArray": {
```

Comment:
Could this be a variable? The containers should be the same through all of the environments.

containers: [
    "container1",
    "container2"
]